www/src/components/guidelines/layout.js

```diff
@@ -36,35 +32,17 @@ const Layout = ({ children, background, pathname, pageTitle }) => (
     />
     <Global styles={globalStyles} />
     <Helmet>
-      <title>
-        {pageTitle ? `${pageTitle} | Guidelines | GatsbyJS` : `GatsbyJS`}
-      </title>
-      <meta name="twitter:site" content="@gatsbyjs" />
-      <meta name="og:type" content="website" />
-      <meta name="og:site_name" content="GatsbyJS" />
```

Comment:
Not sure if this matters, but there is some functional changes in these meta tags that are rendered in the BaseLayout: https://github.com/gatsbyjs/gatsby/blob/b7d34ed058bef76597171490de63a130c23a3baa/www/src/components/site-metadata.js#L28-L43

things that popped out at me is og:site_name is different, the twitter:site is missing, the output will have 2 title fields? unless Helmet resolves them. But you are passing helmet a defaultTitle from the site metadata, but then rendering a title here. I'm not sure what get's priority or how that resolves.

Reply:
* Helmet overrides the title fields with the most specific one. The `defaultTitle` is only shown when <title> isn't defined in a child component
* `og:site_name` resolves to the siteMetadata.title field defined in [Gatsby config](https://github.com/gatsbyjs/gatsby/blob/master/www/gatsby-config.js#L72), which is... GatsbyJS, same as this one.

My interpretation is that these fields were copied over from the base Layout (and thus SiteMetadata) when this page was created, and hard-coded because we didn't want to query siteMetadata here. My guess is that `twitter:site` was added after this page was created, and it wasn't copied over. Having everything use the same site layout gets rid of further errors like this.

Reply:
> Having everything use the same site layout makes sure everything is consistent going forward.

💯

Reply:
> My interpretation is that these fields were copied over from the base Layout (and thus SiteMetadata) when this page was created, and hard-coded because we didn't want to query siteMetadata here. My guess is that twitter:site was added after this page was created, and it wasn't copied over.

I honestly can't remember anymore 😓